pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -262,10 +267,13 @@ func (r *ObjectBucketClaimReconciler) handleDeleteClaim(key client.ObjectKey) er
 	if err != nil || class == nil {
 		return fmt.Errorf("error getting storageclass from OB %q", ob.Name)
```

Comment:
we don't need to get the SC. The OB's reclaim policy should be the authoritative source for reclaim.